### ISSUE TITLE
How to change icon in button dynamically?

### ISSUE THREAD

**Issue by CyberryCJ** [ant-design-blazor/ant-design-blazor#2728]
I tried to change icon in button dynamically. but it wont change. sample code below
![image](https://user-images.githubusercontent.com/36914098/191133838-13ae6749-07ca-42dd-81ff-531f41ab7711.png)
![image](https://user-images.githubusercontent.com/36914098/191133989-0d50ecd9-40dd-4544-b021-e71116e624a0.png)
this image after clicking button. Text changes but button icon not change to unlock icon.
![image](https://user-images.githubusercontent.com/36914098/191134088-7a98da34-723d-489c-8e27-c3c0416ccc3e.png)

**Comment by ElderJames** [comment]
@YongQuan-dotnet cc

**Comment by charset** [comment]
I wrote a sample like your code but it worked as expected.
![image](https://user-images.githubusercontent.com/1499849/195744172-c3b230a7-0b75-48e6-9f16-50b53b33d25c.png)
![image](https://user-images.githubusercontent.com/1499849/195744188-399022be-8c6b-4336-8a27-a610d1c31d30.png)
I'm using the latest version of AntDesignBlazor.
I tested both `@onclick="(async () => await ChangeIcon())"` and `OnClick="ChangeIcon"` working well.
